include/crow/http_connection.h

```diff
@@ -384,7 +386,6 @@ namespace crow
 
         void do_write_static()
         {
-            is_writing = true;
```

Comment:
I'm just curious as to why this was removed, other than that everything seems to be in order

Reply:
The `is_writing` ( and also `is_reading`) variables were used to handle the lifetime of the http_connection. From my understanding, if one of these variables was set to true, then the object was still active and it should not have been deleted. Since now the lifetime is handled from the `shared_ptr`, I thought it was not necessary any more and I remove any reference from the source code. Am I missing something?